### DISCUSSION TITLE
Error 404 in AzuraCast on CyberPanel, Ubuntu and Firefox browser

### DISCUSSION THREAD

**Discussion by joplat** [AzuraCast/AzuraCast#6607] [Q&A]
Hello AzuraCast,

Would someone please help me solve my problem with AzuraCast? Thank you in advance!
Good evening my friend at AzuraCast Radio Software. My issue or problem with AzuraCast is driving me crazy and I have done all I could with YouTube tutorial but all in vain...!! For this reason, I have come to you for help on how to fix the problem or issue I am having.

The error I am having started at the time of my installation of AzuraCast.  At the end of my installation is when I saw an error that looks like the following error message below, but I continued, and tried to use the IP/URL that I received at the end of the installation. Instead, I was greeted with Error 404 on the browser, and I have done a lot to see if I could fix the error 404, but still, I am unable to fix the error 404. After 24 hours I decided to seek for help from GitHub site.

On my machine, I installed CyberPanel Server and OpenLightSpeed Server on "Port 8090." I am hoping that someone will help me diagnose my issue or problem with the "Error 404 Not Found - The resource requested could not be found on this server." Though, after installation it shows that AzuraCast installation is complete. With the fact that it is complete all's good until I tried the URL http://xxx.xxx.x.xxx to be greeted with the Error 404!

Please help me fix this issue that's keeping me from making a progress with AzuraCast. Thanks in advance. 

Regards,
Ody
==========================================

HERE IS PART OF THE ERROR 404 MESSAGE

[OK] AzuraCast installation complete!

      Visit http://xxx.xxx.xx.xx to complete setup.

[+] Running 2/2
 ✔ Container azuracast Created 1.4s
 ✔ Container azuracast_updater Started 1.5s
Error response from daemon: driver failed programming external connectivity on endpoint azuracast (622c6b89bcfe832e16f40997e836631cb88fa9f65a8b0b9ca1e8e10689806fb1): Error starting userland proxy: listen tcp4 0.0.0.0:8090: bind: address already in use
root@server1:/var/azuracast#

**Comment by Piratenkanon** [comment]
The problem will always remain as long as the standard ports are not used.
Port 80 for http
Port 443 for https
is there running also a webserver on the Server with the IP ?  If not, turn off the web server

**Reply by joplat** [reply]
Hello Piratenkanon, 

Thanks for the feedback on my issue with AzuraCast. Yes, I am running OpenLiteSpeed Server on my machine. However, I have turned off the OpenLiteSpeed with hope that I will be able to fix the issue so I could login to the AzuraCast server, but I still run into the error 404 Not Found. The is that I didn't change Ports 80 and 443. There was no way for me to be able change those ports 80 and 443. Do you think I should reload the AzuraCast if not, what else can I do to fix this issue?

Regards, 
Ody.

**Comment by Moonbase59** [comment]
People, the error is _staring in your face_:

> On my machine, I installed CyberPanel Server and OpenLightSpeed Server on "Port 8090."

> Error starting userland proxy: listen tcp4 0.0.0.0:8090: bind: address already in use

**The AzuraCast proxy can’t bind to port 8090 since something is running there already.**

As stated somewhere in the [documentation](https://docs.azuracast.com/), a default AzuraCast install sets up ports for the first 50 stations, i.e. 8000–8499, usually reserving 10 ports per station, and currently using 3 of each block.

Verify by checking `docker-compose.yml` (and `docker-compose.override.yml` if exists) in your host’s `/var/azuracast` folder.

Now you can change all that (see [documentation](https://docs.azuracast.com/)), or simply not run extra services in the 8000–8499 port range.

**Reply by joplat** [reply]
Good morning Moonbase59, 

Your information looks good, but I don't know how to make good use of it. I am unable to make changes through "nano /var/azuracast." When I tried to use nano format to change the port for AzuraCast, However, since I am unable to do so, which other way can this issue be handled, fixed or modified?

Regards, 
Ody

**Reply by joplat** [reply]
Hello Moonbase59,

I tried to change the port number to something else on CyberPanel, but it didn't work for me. Which other idea do you have for me to try please?

Regards,
Ody
